book/website/community-handbook/persona/persona-contributors.md

```diff
@@ -0,0 +1,105 @@
+(ch-persona-contributors)=
+## Contributor's Personas and Pathways
+
+In this document, we have created personas for _The Turing Way_ contributors and users based on their initial engagements with the project as a reader, learner, resource developer, reviewer and maintainer.
+
+The interaction with the project followed by the sustained engagements of personas in a project can be understood in different phases of community membership:
+
+1. Discovery - How they first hear about the project or group.
+2. First Contact - How they first engage with the project or group, their initial interaction.
+3. Participation - How they first participate or contribute.
+4. Sustained Participation - How their contribution or involvement can continue.
+5. Networked Participation - How they may network within the community.
+6. Leadership - How they may take on some additional responsibility on the project, or begin to lead.
+
+We have created personas based on GitHub skills and willingness to contribute to the project.
+We have also listed a few examples of how our personas might experience their first interactions with our project and the "pathway" they take to get involved.
+For each persona, we highlight how their involvement with the Turing Way can improve and enrich the project and community.
+
+This document is meant to suggest ways you might interact with, learn from, and contribute to the Turing Way.
+We hope that one of these personas resonates with you and motivates you to get involved!
+
+## Contributor personas
+
+1. Sam, someone who has no GitHub experience
+2. Alia, someone who has a lot of GitHub experience
+3. Amal, someone who knows they want to contribute and does
+4. Noor, someone who doesn't know they want to contribute, but does
+5. Robin, someone who can only contribute outside of their normal working hours
+
+### 1. Sam, a PhD student with no GitHub experience
+
+Sam is a PhD student in biology at University College London.
+They are learning how to code in Python, but have not yet had any training in version control or GitHub.
+They're interested in learning how to use open notebooks such as Jupyter to do their analysis so that they can easily share their work for critique with their very busy supervisor.
+
+1. **Discovery** - Sam hears about the Turing Way Book Dash events through an email from the department's Open Science Champion.
```

Comment:
```suggestion
1. **Discovery** - Sam hears about _The Turing Way_ Book Dash events through an email from the department's Open Science Champion.
```

Reply:
OMG, how could I get this wrong soo many times!? 🙈